data/mods/DinoMod/dinosaur.json

```diff
@@ -870,11 +860,73 @@
       { "monster" : "mon_giant_crayfish", "freq" :  4, "cost_multiplier" : 0, "starts" : 720, "conditions" : ["DUSK", "NIGHT", "DAWN", "SPRING", "SUMMER", "AUTUMN"] },
       { "monster" : "mon_giant_crayfish", "freq" :  1, "cost_multiplier" : 0, "starts" : 792, "conditions" : ["SPRING", "SUMMER", "AUTUMN"] },
       { "monster" : "mon_giant_crayfish", "freq" :  4, "cost_multiplier" : 0, "starts" : 792, "conditions" : ["DUSK", "NIGHT", "DAWN", "SPRING", "SUMMER", "AUTUMN"] },
+      { "monster" : "mon_bear_smoky", "freq" :  1, "cost_multiplier" : 0, "starts" : 672, "conditions" : ["SPRING", "SUMMER", "AUTUMN"] },
 
```

Comment:
It's still in `data/json/monsters.json` but is also in `data/json/default_blacklists.json`.